### ISSUE TITLE
Searching for items with `unaccent`

### ISSUE THREAD

**Issue by SaeX** [yourlabs/django-autocomplete-light#462]
Since Django 1.8, postgres contrib allows for `unaccent` functionality. For example when `Helene` is searched for, `Hélène` would be returned as well, using `Person.objects.filter(first_name__unaccent=u"Hélene")`

As an enhancement, it would be great if Django-Autocomplete-Light would also allow this possibility as a setting in its registry.

**Comment by jpic** [comment]
Can we have `unaccent__icontains` instead of `__icontains` by default if the database used by the chocies queryset supports unnacent in https://github.com/yourlabs/django-autocomplete-light/blob/master/autocomplete_light/autocomplete/model.py#L129 ?

**Comment by SaeX** [comment]
That'd even be better. Any idea on how to verify whether the DB has `unaccent` support? 
We can't just use `unaccent` for everything as this will fail for DBs that don't have this functionality.

**Comment by jpic** [comment]
True, so we want `unaccent__icontains` instead of `__icontains` if `AUTOMPLETE_LIGHT_ENABLE_UNACCENT` is set or something like that right ?

**Comment by pusateri** [comment]
I'm using this hack for now with the back tick prefix character until the correct way forward emerges:
--- autocomplete/model.py.orig  2015-10-11 00:30:24.000000000 -0400
+++ autocomplete/model.py       2015-10-11 00:44:30.614688720 -0400
@@ -138,6 +138,8 @@
             return "%s__iexact" % field_name[1:]
         elif field_name.startswith('@'):
             return "%s__search" % field_name[1:]
+        elif field_name.startswith('`'):
+            return "%s__unaccent__istartswith" % field_name[1:]
         else:
             return "%s__icontains" % field_name
